src/darwin/list.cpp

```diff
@@ -14,11 +14,15 @@
  * limitations under the License.
  */
 
-#ifndef SRC_DRIVELIST_H_
-#define SRC_DRIVELIST_H_
-
 #include <nan.h>
+#include "../drivelist.hpp"
+
+namespace Drivelist {
 
-NAN_METHOD(list);
+  // TODO(jhermsmeier): Implement
+  std::vector<DeviceDescriptor> ListStorageDevices() {
```

Comment:
Looks like every os `list.cpp` is the same. Can we make them all one file?

Reply:
The Linux & Mac OS `list.cpp` files are functional stubs, so that there's nothing that can go boom on those platforms, while also facilitating a handy starting point for their coming implementations; so I'd keep them.